pages/pages.go

```diff
@@ -207,65 +210,114 @@ func toHTML(isReception bool, filename string) (string, error) {
-		metaPath    = filepath.Join(dir, "meta.toml")
+	const (
+		bodyPattern     = "body(_\\w+)\\.md"
+		sidebarPattern  = "sidebar(_\\w+)\\.md"
+		titlePattern    = "Title(_\\w+)"
+		metaFile        = "meta.toml"
+		iso8601DateTime = "2006-01-02T15:04:05Z"
 	)
 
-	// Parse markdown to HTML.
-	body, err := toHTML(isReception, bodyPath)
-	if err != nil {
-		return nil, err
-	}
-	log.WithField("body", body).Debug("HTML of body.md")
+	bodyReg := regexp.MustCompile(bodyPattern)
+	sidebarReg := regexp.MustCompile(sidebarPattern)
+	titleReg := regexp.MustCompile(titlePattern)
 
-	// Parse sidebar to HTML.
-	sidebar, err := toHTML(isReception, sidebarPath)
-	if err != nil {
-		return nil, err
-	}
-	log.WithField("sidebar", sidebar).Debug("HTML of sidebar.md")
+	bodies := make(LangLookup)
+	sidebars := make(LangLookup)
+	commitTimes := make(LangLookup)
+	titles := make(LangLookup)
+	anchorsLists := make(LangAnchorLookup)
 
-	// Get commit time
-	commitTime, err := getCommitTime(root, bodyPath)
-	if err != nil {
-		return nil, err
-	}
+	entries, err := os.ReadDir(dir)
 
-	// Parse anchors in the body.
-	anchs, err := anchor.Anchors(body)
-	if err != nil {
-		return nil, err
+	for _, entry := range entries {
+		if entry.IsDir() {
+			continue
+		}
+
+		entryPath := filepath.Join(dir, entry.Name())
+
+		if match := bodyReg.FindSubmatch([]byte(entry.Name())); match != nil {
+			lang := ""
+			if len(match) > 1 && len(match[1]) > 0 {
+				lang = string(match[1][1:])
+			}
+			bodies[lang], err = toHTML(isReception, entryPath)
+			log.WithField("body", bodies[lang]).Debug("HTML of body_" + lang + ".md")
+
+			if err != nil {
+				return nil, err
+			}
+
+			commitTime, err := getCommitTime(root, entryPath)
+			if err != nil {
+				commitTimes[lang] = time.Now().Format(iso8601DateTime)
+			} else {
+				commitTimes[lang] = commitTime.Format(iso8601DateTime)
+			}
+
+			// Parse anchors in the body.
+			anchorsLists[lang], err = anchor.Anchors(bodies[lang])
+			if err != nil {
+				return nil, err
+			}
+		}
+
+		if match := sidebarReg.FindSubmatch([]byte(entry.Name())); match != nil {
+			lang := ""
+			if len(match) > 1 && len(match[1]) > 0 {
+				lang = string(match[1][1:])
+			}
+			sidebars[lang], err = toHTML(isReception, entryPath)
+			log.WithField("sidebar", sidebars[lang]).Debug("HTML of sidebar" + lang + ".md")
+			if err != nil {
+				return nil, err
+			}
+		}
 	}
 
 	// Parse meta data from a toml file.
+	metaPath := filepath.Join(dir, metaFile)
 	var meta = Meta{
 		Sort:     nil, // all pages without a sort-tag should be after the pages with a sort-tag, but should keep their internal order
 		Expanded: false,
 	}
+	var metaMap = make(map[string]any)
 	if _, err := toml.DecodeFile(metaPath, &meta); err != nil {
 		return nil, err
 	}
+	if _, err := toml.DecodeFile(metaPath, &metaMap); err != nil {
+		return nil, err
+	}
 
 	if meta.Sensitive && isReception {
 		return nil, nil
 	}
+	for k, v := range metaMap {
```

Comment:
Instead of this, maybe it would be nicer to have the toml file contain an object called titles that contains a key for each language. In toml syntax that could be written as:
```toml
Sort = 7
[Title]
sv = "Nämnder"
en = "Clubs"
```

Reply:
Great idea!